src/test/java/io/jenkins/plugins/analysis/core/charts/NewVersusFixedTrendChartTest.java

```diff
@@ -0,0 +1,69 @@
+package io.jenkins.plugins.analysis.core.charts;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.assertj.core.util.Lists;
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.Severity;
+
+import io.jenkins.plugins.analysis.core.charts.ChartModelConfiguration.AxisType;
+import io.jenkins.plugins.analysis.core.model.AnalysisResult.BuildProperties;
+import io.jenkins.plugins.analysis.core.util.AnalysisBuild;
+import io.jenkins.plugins.analysis.core.util.LocalizedSeverity;
+import io.jenkins.plugins.analysis.core.util.StaticAnalysisRun;
+
+import static java.util.Collections.*;
+import static net.javacrumbs.jsonunit.assertj.JsonAssertions.*;
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+
+/**
+ * Tests the class {@link NewVersusFixedTrendChart}.
+ *
+ * @author Fabian Janker
+ */
+
+class NewVersusFixedTrendChartTest {
+
+    @Test
+    void shouldCreateALinesChartModel() {
+        NewVersusFixedTrendChart chart = new NewVersusFixedTrendChart();
+
+        List<StaticAnalysisRun> results = new ArrayList<>();
+        results.add(createResult(1));
+        results.add(createResult(2));
+
+        LinesChartModel model = chart.create(results, createConfiguration());
+
+        verifySeries(model.getSeries().get(0), Palette.RED);
+        verifySeries(model.getSeries().get(1), Palette.GREEN);
+
```

Comment:
Also add assertions for the y-axis values, see [SeverityTrendChart as an example](https://github.com/jenkinsci/warnings-ng-plugin/blob/master/src/test/java/io/jenkins/plugins/analysis/core/charts/SeverityTrendChartTest.java#L40).